solvcon/parcel/vewave/solver.py

```diff
@@ -118,6 +118,7 @@ def __init__(self, blk, mtrldict, **kw):
         self.stm = np.empty((ngstcell+ncell, neq), dtype=fpdtype)
         self.cfl = np.empty(ngstcell+ncell, dtype=fpdtype)
         self.ocfl = np.empty(ngstcell+ncell, dtype=fpdtype)
+        from solvcon.parcel.vewave import _algorithm
```

Comment:
You do not need this local import if line 53 works:

```python
sc.import_module_may_fail('._algorithm')
```

Please fix `import_module_may_fail`.